### PR TITLE
git ls-files -z -- .pre-commit-config.yaml | xargs -0 sed -i 's#https://gitlab.com/PyCQA/flake8#https://github.com/PyCQA/flake8#'

### PR DESCRIPTION
**Pull request by VannTen**
Committed via https://github.com/asottile/all-repos

### CODE REVIEW

**Comment by VannTen** [comment]
looks like the old pre-commit job from .aicoe-ci.yaml sticks around. Does that needs a manual merge ? @goern

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 check:
-  - thoth-precommit
   - thoth-build
 build:
-  base-image: "quay.io/thoth-station/s2i-thoth-f32-py38-cloc:latest"
+  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.35.0
```
Comment (harshad16) [inline .aicoe-ci.yaml:4]: please revert this change 
ubi doesn't have cloc base packages.

if this package can be installed in ubi
https://github.com/thoth-station/si-cloc/blob/cea233837f6209878f92ccc51270ccba2f9da13f/Dockerfile#L4
then please lets update the Dockerfile.

**Review by harshad16 — CHANGES_REQUESTED** [review]
Change is needed.

File: .aicoe-ci.yaml (hunk 1 above)
Comment (VannTen) [reply .aicoe-ci.yaml:4]: So what's the preferred option, installing cloc in the Dockerfile or using the cloc base image ?
@goern @codificat

Comment (codificat) [reply .aicoe-ci.yaml:4]: well... I suggested the image update when I saw "f32" in the current one. It is 2 years old and it will only keep getting older. So, not an ideal option IMHO.

But right: there's no `cloc` in the UBI repos, nor in EPEL... so installing it in UBI is probably bound to be not ideal either?

As a quick solution, maybe for the time being we can stick to the current image and accept its aging?

**Comment by VannTen** [comment]
Works for me.

**Comment by VannTen** [comment]
How can we bypass the pre-commit check ? prow/pre-commit should have a more up to date version..
@harshad16  @goern  ?

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve

Thanks :100:

**Comment by harshad16** [comment]
> How can we bypass the pre-commit check ? 

It was removed from .aicoe-ci.yaml file
so it would not start from next pull request onwards.
